src/pages/Admin/Roles/RoleForm.tsx

```diff
@@ -27,12 +29,35 @@ export default function RoleForm({
 }: RoleFormProps) {
   const { t } = useTranslation();
   const queryClient = useQueryClient();
-  const [formData, setFormData] = React.useState({
-    name: role?.name || "",
-    description: role?.description || "",
-    permissions: role?.permissions.map((p) => p.slug) || [],
+
+  const roleSchema = z.object({
+    name: z.string().trim().min(1, t("name_required")),
```

Comment:
```suggestion
    name: z.string().trim().min(1, t("name_is_required")),
```

key is already present for this value